test/configuration.test.ts

```diff
@@ -0,0 +1,91 @@
+import { expect } from 'chai';
+import * as vscode from 'vscode';
+import * as cp from 'child_process';
+import * as pq from 'proxyquire';
+
+const childProcessStub: any = {};
+const extensionConfig = pq('../src/configuration', {
+  'child_process': childProcessStub,
+});
+
+const { RubocopConfig, getConfig } = extensionConfig;
+const canFindBundledCop = () => new Buffer('path/to/bundled/rubocop');
+const cannotFindBundledCop = () => { throw new Error('not found'); };
+
+describe('RubocopConfig', () => {
+  // let config: RubocopConfig;
+
+  describe('getConfig', () => {
+    describe('.useBundler', () => {
+      it('is set to true if a bundled rubocop is found', () => {
+        childProcessStub.execSync = canFindBundledCop;
+        expect(getConfig()).to.have.property('useBundler', true);
+      });
+
+      it('is unset if a bundled rubocop is not found', () => {
+        childProcessStub.execSync = cannotFindBundledCop;
+        expect(getConfig()).to.have.property('useBundler', undefined);
+      });
+
+    });
+
+    describe('.command', () => {
+      describe('win32 platform', () => {
+        beforeEach(() => {
+          this.originalPlatform = Object.getOwnPropertyDescriptor(process, 'platform');
+          Object.defineProperty(process, 'platform', {
+            value: 'win32',
+          });
+        });
+        afterEach(() => {
+          Object.defineProperty(process, 'platform', this.originalPlatform);
+        });
+
+        it('is set to "bundle exec rubocop.bat" if bundled rubocop is present', () => {
+          childProcessStub.execSync = canFindBundledCop;
+          expect(getConfig()).to.have.property('command', 'bundle exec rubocop.bat');
+        });
+
+        it('is set to "rubocop.bat" otherwise', () => {
+          childProcessStub.execSync = cannotFindBundledCop;
+          expect(getConfig()).to.have.property('command', 'rubocop.bat');
```

Comment:
Another case where i'm not sure what this would actually look like. Would it be a fully-qualified windows fs path? I don't know. Sorry.

Reply:
This may actually be a good use-case for setting up windows build hosts in CI, if that option is available. Probably beyond the scope of this PR, though. I don't think you'll be able to verify this as-is without somebody handy with a windows machine that includes a ruby dev environment.

Reply:
I did find this, but it makes me think that this is more complicated than I initially assumed:
https://github.com/sirlantis/rubocop-for-rubymine/issues/9